lib/git_reflow.rb

```diff
@@ -27,12 +27,11 @@ def status(destination_branch)
     pull_request = git_server.find_open_pull_request( :from => current_branch, :to => destination_branch )
 
     if pull_request.nil?
-      puts "\n[notice] No pull request exists for #{current_branch} -> #{destination_branch}"
-      puts "[notice] Run 'git reflow review #{destination_branch}' to start the review process"
+      say "\nNo pull request exists for #{current_branch} -> #{destination_branch}", :notice
+      say "Run 'git reflow review #{destination_branch}' to start the review process", :notice
     else
-      puts "Here's the status of your review:"
+      say "Here's the status of your review:"
```

Comment:
I just wanted it to be in a position where all we need to do is update the `say` command.  The path forward is pretty clear now.